### PR TITLE
Assorted LXQt components

### PR DESCRIPTION
**Pull request by ifreund**
Someone has reported issues with lxqt-panel and lxqt-runner under river and I think I need to run them under a debugger to figure out what's going on. Unfortunately these packages don't work yet and seem to require further LXQt components in order to function. I guess I'll keep packaging LXQt stuff until I get them running properly or get bored.

### CODE REVIEW

**Comment by ifreund** [comment]
This appears to be enough to test the things I wanted to test. I don't plan on packinging further LXQt components at this time. What's here should already be quite useful for anyone wanting to take this further.

File: contrib/muparser/template.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+pkgname = "muparser"
+pkgver = "2.3.4"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DENABLE_OPENMP=OFF"]
```
Comment (nekopsykose) [inline contrib/muparser/template.py:5]: this just needs libomp-devel in makedepends and then it works (most likely)

File: contrib/qtxdg-tools/template.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "lxqt-build-tools",
+]
+makedepends = [
+    "libqtxdg-devel",
+]
+pkgdesc = "User tools for libqtxdg"
+maintainer = "Isaac Freund <mail@isaacfreund.com>"
+license = "LGPL-2.1-or-later"
+url = "https://github.com/lxqt/qtxdg-tools"
+source = f"{url}/releases/download/{pkgver}/{pkgname}-{pkgver}.tar.xz"
```
Comment (nekopsykose) [inline contrib/qtxdg-tools/template.py:18]:
```suggestion
source = f"{url}/releases/download/{pkgver}/qtxdg-tools-{pkgver}.tar.xz"
```

(for everything)

File: contrib/lxqt-panel/template.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+pkgname = "lxqt-panel"
+pkgver = "2.0.1"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCPULOAD_PLUGIN=NO",
+    "-DNETWORKMONITOR_PLUGIN=NO",
+    "-DSYSSTAT_PLUGIN=NO",
```
Comment (nekopsykose) [inline contrib/lxqt-panel/template.py:8]: any reason to disable these

File: contrib/lxqt-panel/template.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "layer-shell-qt-devel",
+    "libdbusmenu-lxqt-devel",
+    "liblxqt-devel",
+    "libsensors-devel",
+    "lxqt-globalkeys-devel",
+    "lxqt-menu-data",
+    "qt6-qttools-devel",
+    "solid-devel",
+    "xcb-util-devel",
+    "libpulse-devel",
```
Comment (nekopsykose) [inline contrib/lxqt-panel/template.py:30]:
```suggestion
    "kwindowsystem-devel",
    "layer-shell-qt-devel",
    "libdbusmenu-lxqt-devel",
    "liblxqt-devel",
    "libpulse-devel",
    "libsensors-devel",
    "lxqt-globalkeys-devel",
    "lxqt-menu-data",
    "qt6-qttools-devel",
    "solid-devel",
    "xcb-util-devel",
```

(sort all the lists)

File: contrib/muparser/template.py (hunk 1 above)
Comment (ifreund) [reply contrib/muparser/template.py:5]: Honestly, I really don't think a parsing library that is only used to implement a calculator in the `lxqt-runner` program needs multithreading...

If enabling it doesn't make things more fragile I guess it wouldn't hurt though.

File: contrib/qtxdg-tools/template.py (hunk 2 above)
Comment (ifreund) [reply contrib/qtxdg-tools/template.py:18]: sure, the status quo did make copy pasting to create these templates easier though.

File: contrib/lxqt-panel/template.py (hunk 3 above)
Comment (ifreund) [reply contrib/lxqt-panel/template.py:8]: The reason is that I was too lazy to package the required dependencies

File: contrib/muparser/template.py (hunk 1 above)
Comment (nekopsykose) [reply contrib/muparser/template.py:5]: if they have it on by default then may as well leave it on

File: contrib/lxqt-themes/template.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "lxqt-build-tools",
+    "ninja",
+    "perl",
+    "pkgconf",
+]
+makedepends = []
+depends = []
```
Comment (nekopsykose) [inline contrib/lxqt-themes/template.py:13]:
```suggestion
```

File: contrib/muparser/template.py (hunk 1 above)
Comment (ifreund) [reply contrib/muparser/template.py]: done

**Pull request merged**